src/services/me/index.js

```diff
@@ -1,5 +1,6 @@
 const { Forbidden, GeneralError } = require('@feathersjs/errors');
 const logger = require('../../logger');
+const { userModel } = require('../../services/user/model');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['userModel' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5229889)